streams/us.m3u

```diff
@@ -1947,3 +1947,225 @@ https://2-fss-2.streamhoster.com/pl_120/amlst:203530-1426406/playlist.m3u8
+https://jlt2104.cdn.nextologies.com/ebaed9568f2b8e61/ffaaf4b7e1d0e7a8/ea5f88d4e7c252b2/056b5eafae61b1e6/playlist.m3u8
+#EXTINF:-1 tvg-id="JewelryTVExtra.us" status="",Jewelry TV Extra (720p)
+https://cdn3.wowza.com/1/R3ZYNjNvdGVoaDFZ/Q3pENnlF/hls/live/playlist.m3u8
+#EXTINF:-1 tvg-id="DCCouncilHearingRoom120.us" status="",DC.gov Council Hearing Room 120 (Washington DC) (1080p)
+https://video.oct.dc.gov/out/u/97_12.m3u8
+#EXTINF:-1 tvg-id="DCC.us" status="",DC.gov DCC (Washington DC) (1080p)
+https://video.oct.dc.gov/out/u/DCC.m3u8
+#EXTINF:-1 tvg-id="DKN.us" status="",DC.gov DKN (Washington DC) (1080p)
+https://video.oct.dc.gov/out/u/DKN.m3u8
+#EXTINF:-1 tvg-id="DoverTV.us" status="",Dover TV (Dover DE) (480p)
+https://dover-de.secure.footprint.net/egress/bhandler/doverde/streama/playlist.m3u8
+#EXTINF:-1 tvg-id="WITN22.us" status="",WITN 22 (Wilmington DE) (1080p) [Not 24/7]
+https://edge-f.swagit.com/live/wilmingtonde/smil:std-4x3-1-a/playlist.m3u8
+#EXTINF:-1 tvg-id="NCCTV.us" status="",NCCTV (New Castle County DE) (1080p)
+https://newcastlecoutv.secure.footprint.net/egress/bhandler/newcastlecou/streama/playlist.m3u8
+#EXTINF:-1 tvg-id="GreenbeltTV.us" status="",Greenbelt TV (Greenbelt MD) (480p)
+https://greenbeltacctv.secure.footprint.net/egress/bhandler/greenbelt/streama/playlist.m3u8
+#EXTINF:-1 tvg-id="ItsSupernaturalNetwork.us" status="",It's Supernatural! Network (ISN Network) (1080p)
+https://isntv.mmdlive.lldns.net/isntv/f79b65204e7141c6a5cc74e63cf0dae5/playlist.m3u8
+#EXTINF:-1 tvg-id="IndTVUSA.us" status="",Ind TV USA (720p)
+https://indtv.secure.footprint.net/egress/bhandler/indtv/streama/playlist.m3u8
+#EXTINF:-1 tvg-id="GreensboroTV.us" status="",Greensboro TV (Greensboro NC) (720p)
+https://lin12.isilive.ca/live/greensboro/GTN/index.m3u8
+#EXTINF:-1 tvg-id="WLCT96.us" status="",Wolcott Governmental TV (720p) (Wolcott CT)
+https://5a5c57d042315.streamlock.net/live11704001/ngrp:government_all/playlist.m3u8
+#EXTINF:-1 tvg-id="CommunityTVChannel189.us" status="",Traverse Area Community Media CommunityTV Channel 189 (Traverse City MI) (1080p) [Geo-blocked]
+https://reflect-tacm.cablecast.tv/live-3/live/live.m3u8
+#EXTINF:-1 tvg-id="WPTV.us" status="",WPTV (540p) (Westport CT)
+https://securestream3.champds.com/hlssstc/WestportCTLIVE/WestportCTLIVE.m3u8
+#EXTINF:-1 tvg-id="WPAATV.us" status="",WPAA TV (1080p)
+https://streams.helnix.com/autoHLS/ce3a40274f01c8fc/ce3a40274f01c8fc.m3u8
+#EXTINF:-1 tvg-id="RockyHillChannel16.us" status="",Rocky Hill Government Access Channel 16 (Rocky Hill CT) (540p)
+https://securestream3.champds.com/hlssstc/RockyHillCTLIVE/RockyHillCTLIVE.m3u8
+#EXTINF:-1 tvg-id="GodsLearningChannel.us" status="",GLC TV God's Learning Channel (480p)
+https://2-fss-1.streamhoster.com/pl_122/200562-1176456-1/playlist.m3u8
+#EXTINF:-1 tvg-id="GlobalBuddhistNetwork.us" status="",GBN Global Buddhist Network (1080p)
+https://us.gbnus.com/live/dhamma_high/playlist.m3u8
```

Comment:
Going to add this one over IPTV_Exception, as this one contains different qualities but no adaptive stream.

Reply:
Please check https://github.com/LaneSh4d0w/IPTV_Exception/raw/master/channels/us/gbnus.m3u8 and tell me if you can see both High and Low.

Reply:
> Please check https://github.com/LaneSh4d0w/IPTV_Exception/raw/master/channels/us/gbnus.m3u8 and tell me if you can see both High and Low.

Yes I can see both high and low streams.

Reply:
Perfect, then
```suggestion
https://github.com/LaneSh4d0w/IPTV_Exception/raw/master/channels/us/gbnus.m3u8
```